.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
+
+language: bash
+
+services:
+  - docker
+
+before_script:
+  - export APP_IMAGE=keboola-component
```

Comment:
Nevím, jestli jsme to už neřešili, ale pro-forma bych tady psal keboola/s3-writer

Reply:
uteklo, díky!

Reply:
tohle by ideálně měl celý vyřešit ten generator

Reply:
Potřebuje to nejdřív templatování v generátoru https://github.com/keboola/component-generator/issues/6